domclone/storybook
# Parallelize CI

Split both the `examples` and `cli-test` files for faster build times.

Added `parallelism: 4` as a dependency. `4` is arbitrary and can be updated if need be.

## What I did
I added `parallelism` key and set to `4` (which is arbitrary and can be updated).
I added a command to split the `examples` test files.
I added a command to split the `cli-test` test files.

## How to test
Check that build times for `examples` (12:56)  and `cli-test` (21:47) jobs are faster than what is 

- Is this testable with Jest or Chromatic screenshots? (No)
- Does this need a new example in the kitchen sink apps? (No)
- Does this need an update to the documentation? (No)
